src/api/app/views/webui2/webui/groups/_group_members.html.haml

```diff
@@ -25,11 +25,10 @@
                   class: 'group-member-removal float-right', title: 'Remove member from group') do
                   %i.fas.fa-sm.fa-times-circle.text-danger
               .card-body.ml-2
-                %b
-                  = link_to(user_show_path(user)) do
-                    = display_name(user)
+                = link_to(user_show_path(user), class: 'user-realname') do
+                  = display_name(user)
 
-                .custom-control.custom-checkbox
+                .custom-control.custom-checkbox.mt-3
```

Comment:
I believe `.mt-3` isn't needed.

Reply:
I believe we really need some more space between the name and the roles list. Maybe `mt-2` is enough. See the difference:

**No margin:**
![Screenshot-2019-8-20 Open Build Service](https://user-images.githubusercontent.com/2581944/63331015-14228380-c335-11e9-8c0b-1e7f767893ad.png)

**mt-2:**
![Screenshot-2019-8-20 Open Build Service(1)](https://user-images.githubusercontent.com/2581944/63331029-1d135500-c335-11e9-96ea-a804d46bd2ea.png)

**mt-3:**
![Screenshot-2019-8-20 Open Build Service(2)](https://user-images.githubusercontent.com/2581944/63331049-269cbd00-c335-11e9-804d-636623cb3bf0.png)

Reply:
I prefer `mt-2` or even `mt-1` would be enough in this case.